ansys/api/dbu/v0/dbuapi.proto

```diff
@@ -31,9 +31,10 @@ message SetVersionResponse{
 
 enum ApiVersion {
   INVALID = 0;
-  V_21 = 21;
+  V_21 = 21; //DEPRECATED
```

Comment:
Why can't this be removed, due to retro compatibility?

Reply:
Version 21 still needs to be available for retro compatibility - but we should use the latest released version from now onwards